bbb-install-2.6.sh

```diff
@@ -155,9 +155,6 @@ main() {
       l)
         LETS_ENCRYPT_ONLY=true
         ;;
-      g)
-        GREENLIGHT=true
```

Comment:
Did you mean to remove the other block below instead? The API_DEMOS=true

I believe we still want to keep GreenLight installation optional. The fact that we remove `-a` does not mean we should default to `-g`

Likely we want some kind of log or even preventing progress if `-a` was passed? What do you think?

Reply:
I will return the `-g` option and remove the `-a` solely.

> Likely we want some kind of log or even preventing progress if -a was passed? What do you think?

I think logging a warning would be nice, with the message: "Installing Demo-API is not possible anymore". Does that sound good?